tools/reactive-controllers/test/downstate.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { expect, fixture, nextFrame } from '@open-wc/testing';
+import { html, LitElement } from 'lit';
+import { DownState } from '@spectrum-web-components/reactive-controllers/src/downstate.js';
+
+class TestDownstateEl extends LitElement {}
```

Comment:
Can we use `spectrumConfig` here to ensure both the Controller and the delegates approach?

Reply:
May also want to wrap it all in an `<sp-theme>` to get ALL of the contracts covered.

Reply:
```js
class TestDownstateEl extends SpectrumElement {
    override spectrumConfig = {
        downstate: ['spectrum-two'],
    };
    protected override render(): TemplateResult {
        return html`
            <sp-theme system="spectrum-two" color="light" scale="medium">
                <slot></slot>
            </sp-theme>
        `;
    }
}
```